### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by elizabethmcd**
This PR has several small fixes, including: 
- Adds missing `systemfonts` dependency #26 
- `primary_ordered` is the default palette #24 
- Axis/ticks weights are now correct, as measured within Illustrator. Some of these numbers within R are arbitrary so I had to figure out the ratio to pts/inches and manually check in Illustrator it was correct #23 
- Gradient positions for colors should be fixed now and carried through, as they weren't being carried through to the plots before

The next PR(s) will have two larger fixes/additions: 
- Custom ggsave() function to specify our exports to adhere to the Illustrator templates
- Rework of font loading, because there is an issue with the fonts not being embedded upon export and therefore not editable in Illustrator, which is a very large problem. This might be integrated into the custom ggsave() addition but uncertain as I've still been exploring potential solutions.

### CODE REVIEW

File: R/arcadia.R [hunk 1]
```diff
@@ -40,8 +40,8 @@ theme_arcadia <- function(x_axis_type = "numerical", y_axis_type = "numerical",
   y_axis_size <- if (y_axis_type == "categorical") 15 else 14.5
 
   # axis ticks
-  x_axis_ticks <- if (x_axis_type == "categorical") ggplot2::element_blank() else ggplot2::element_line(color="black", size = 0.5)
-  y_axis_ticks <- if (y_axis_type == "categorical") ggplot2::element_blank() else ggplot2::element_line(color="black", size = 0.5)
+  x_axis_ticks <- if (x_axis_type == "categorical") ggplot2::element_blank() else ggplot2::element_line(color="black", size = 0.35) # size ratio to pts is 2.13, so .75 in pts is .35 here
```
Comment (mezarque) [inline R/arcadia.R:43]: So crazy that you had to use post-it math to figure this out! Thank you for doing that!

File: R/arcadia.R [hunk 2]
```diff
@@ -69,7 +69,9 @@ theme_arcadia <- function(x_axis_type = "numerical", y_axis_type = "numerical",
     # tick specifications
     axis.ticks.x = x_axis_ticks,
     axis.ticks.y = y_axis_ticks,
-    axis.line = ggplot2::element_line(color="black", size = 0.5),
+    axis.ticks.length.x = ggplot2::unit(0.07, "in"), # 5 pixels is roughly 0.05 inches
```
Comment (mezarque) [inline R/arcadia.R:72]: Just curious, is the comment or the value out of date?

```suggestion
    axis.ticks.length.x = ggplot2::unit(0.07, "in"), # 5 pixels is roughly 0.07 inches
```

**Review by mezarque — APPROVED** [review]
Thanks for resolving these bugs! The gradients and spacing look much better now.

File: R/arcadia.R (hunk 2 above)
Comment (elizabethmcd) [reply R/arcadia.R]: oh yes, thanks for catching!

**Pull request merged**